site/source/tutorials/1045_custom_elements_in_dojo1/index.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: tutorial
+title: Using Dojo 2 Widgets in a Dojo 1 Application
+overview: Find out how to use Dojo 2 Widgets in a Dojo 1 application.
+paginate: true
+---
+
+{% section 'first' %}
+
+# Using Dojo 2 Widgets in a Dojo 1 Application
```

Comment:
Using Dojo 2 Widgets in a Dojo 1 Application -> Using Dojo 2 widgets in a Dojo 1 application